src/lib/types.ts

```diff
@@ -48,9 +48,14 @@ export interface BeaconInformation {
 export interface Vessel {
   maxCapacity: string;
   vesselName: string;
+  beaconLocation: string;
+  pln: string;
```

Comment:
I agree with your instincts on `pln` as a name.  I think `portLetterNumber` is OK, not too long and in keeping with other properties on this interface (e.g. `rigPlatformLocation` and `officialNumber`).